frontend/src/pages/ErrorsV2/ErrorTag/ErrorTag.tsx

```diff
@@ -33,31 +33,23 @@ const ErrorTag = React.memo(
 					kind="secondary"
 					size="medium"
 					shape="basic"
-					lines="1"
```

Comment:
I think the `lines="1"` was added here incase the tag text was really long. Do we know if a long tag text will cause a line break or not?

Reply:
Yeah, I suppose that could technically be something longer. Just added this back and truncated the error tag `Tag` as well. The text clipping is still fixes here - the problem was with the `<Text>` element we were adding because `lines="1"` was being passed to the `<Text>` inside of `Tag` but not applied on the one we added in this component.